sequencer/src/lib.rs

```diff
@@ -304,18 +289,32 @@ async fn init_hotshot<N: network::Type>(
     .unwrap()
 }
 
+pub struct NetworkParams {
+    pub da_server_url: Url,
+    pub consensus_server_url: Url,
+    pub orchestrator_url: Url,
+}
+
 pub async fn init_node(
-    addr: SocketAddr,
+    network_params: NetworkParams,
     genesis_block: Block,
-    metrics: Box<dyn Metrics>,
-) -> (
-    SystemContextHandle<SeqTypes, Node<network::Centralized>>,
-    u64,
-) {
-    let (config, _, networking) =
-        CentralizedServerNetwork::connect_with_server_config(metrics, addr).await;
-    let da_channel = CentralizedCommChannel::new(Arc::new(networking.clone()));
-    let quorum_channel = CentralizedCommChannel::new(Arc::new(networking.clone()));
+    _metrics: Box<dyn Metrics>,
+) -> (HotShotHandle<SeqTypes, Node<network::Web>>, u64) {
+    // Orchestrator client
+    let validator_args = ValidatorArgs {
+        host: network_params.orchestrator_url.host().unwrap().to_string(),
+        port: network_params.orchestrator_url.port().unwrap(),
+        public_ip: None,
+    };
+    let orchestrator_client = OrchestratorClient::connect_to_orchestrator(validator_args).await;
+    let public_ip = IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1));
```

Comment:
In what sense is this IP "public", or how does it identify the node? If all nodes are using 127.0.0.1?

Reply:
I believe this is only used for the libp2p setup, which we aren't using here so this can pretty much be anything, I can add a comment explaining that